src/N3Store.js

```diff
@@ -277,8 +276,7 @@ export default class N3Store {
   // ### `has` determines whether a dataset includes a certain quad.
   // Returns true or false as appropriate.
   has(quad) {
-    const quads = this.getQuads(quad.subject, quad.predicate, quad.object, quad.graph);
-    return quads.length !== 0;
+    return !this._yieldQuads(quad.subject, quad.predicate, quad.object, quad.graph).next().done;
```

Comment:
I see an opportunity to allow for `has(subject, predicate, object, graph)` here that accepts wildcards.

Reply:
In ecd676488e15fe9834fdebbf49c04b4a50dce1f2.